<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModTax</class>
 <widget class="QMainWindow" name="ModTax">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QRadioButton" name="isBasic">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>60</y>
      <width>84</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Basic product</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="isNormal">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>60</y>
      <width>101</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Normal product</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="newTax">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>180</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>150</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>New tax</string>
    </property>
   </widget>
   <widget class="QPushButton" name="confirmNewTax">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>270</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Confirm</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
